server/http_status.go

```diff
@@ -193,6 +193,35 @@ func (s *Server) startHTTPServer() {
 	serverMux.HandleFunc("/debug/pprof/profile", tracecpu.ProfileHTTPHandler)
 	serverMux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	serverMux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+	ballast := make([]byte, 0)
+	serverMux.HandleFunc("/debug/ballast-object-sz", func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case http.MethodGet:
+			_, err := w.Write([]byte(strconv.Itoa(len(ballast))))
+			terror.Log(err)
+		case http.MethodPost:
```

Comment:
Posting with 0 size means deleting it?

Reply:
Yes. Then the ballast object would be a zero-len and zero-cap byte slice. The previous ballast object would be garbage-collected when the next GC is triggered.